### DISCUSSION TITLE
[hints.md] Support changing URL through user browser language

### DISCUSSION THREAD

**Discussion by F-park** [laike9m/Python-Type-Challenges#85] [Ideas]
## Description

Python's docs support multi languages.
![image](https://github.com/laike9m/Python-Type-Challenges/assets/52167622/14dafda9-fef3-41d9-9f85-bf963b15e95a)

## Idea

User browser language can get by [Navigator.language](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/language)

I consider the hint's URL can change  as below.
`https://docs.python.org/3/library/typing.html#the-any-type`  ->
`https://docs.python.org/zh-cn/3/library/typing.html#the-any-type`

**Comment by laike9m** [comment]
Unfortunately, the Chinese doc (and perhaps other translations) can sometimes be outdated, and I personally don't want to make a decision for the users -- they can always switch themselves if needed.
